src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/MenyvalgFagsak.tsx

```diff
@@ -35,11 +37,13 @@ const MenyvalgFagsak = ({ bruker, minimalFagsak }: IProps) => {
                     brevmottakere={manuelleBrevmottakerePåFagsak}
                 />
             ) : (
-                <Dropdown.Menu.List.Item
-                    onClick={() => navigate(`/fagsak/${minimalFagsak.id}/dokumentutsending`)}
-                >
-                    Send informasjonsbrev
-                </Dropdown.Menu.List.Item>
+                erSaksbehandlerEllerHøyere && (
```

Comment:
Ser at togglen på linje 34 kan saneres, den har vært på i prod siden januar. Da kan vi også bygge om fra ternary til separate and-conditions for å rydde litt i logikktreet. Pseudokode:
```
{erPåDokumentutsending && (<LeggTilEllerFjernBrevmottakere ... />}
{erSaksbehandlerEllerHøyere && (<Dropdown.Menu.List.Item ... />}
```

Reply:
Se diskusjon på slack angående togglen https://nav-it.slack.com/archives/C0205N5B9CK/p1716451904763429?thread_ts=1716277968.812089&cid=C0205N5B9CK

Reply:
Fikset i [Fjern ubrukt toggle og splitt opp i separate and conditions for å ryd…](https://github.com/navikt/familie-ba-sak-frontend/pull/3172/commits/669467da7f380c36325851578b3a3b56f6d004d2)